apps/server/src/modules/authentication/controllers/login.controller.ts

```diff
@@ -59,13 +59,13 @@ export class LoginController {
 	@ApiResponse({ status: 400, type: ValidationError, description: 'Request data has invalid format.' })
 	@ApiResponse({ status: 403, type: ForbiddenOperationError, description: 'Invalid user credentials.' })
 	async loginOauth2(
-		@CurrentUser() user: ICurrentUser,
+		@CurrentUser() user: OauthCurrentUser,
 		// eslint-disable-next-line @typescript-eslint/no-unused-vars
 		@Body() _: Oauth2AuthorizationBodyParams
 	): Promise<LoginResponse> {
 		const loginDto: LoginDto = await this.loginUc.getLoginData(user);
```

Comment:
I think we need a mapper or some sort of safety mechanism here, because you currently put the externalIdToken into the SVS JWT. You should also add an api test that this is not happening.

Reply:
Totally right. Thanks!